### PR TITLE
*: update and fix benchmark script

### LINKED ISSUE #1202: nsqd: ec2 benchmark is broken
bench/bench.py seems to follow the old build pipeline

### PR DESCRIPTION
**Pull request by mreiferson**
replaces #1204
fixes #1202

cc @martingrambow @ploxiln

### CODE REVIEW

File: bench/bench.py [hunk 1]
```diff
@@ -250,30 +257,22 @@ def run():
                      max_duration / total_ops * 1000 * 1000)
 
     for ssh_client, chan in nsqd_chans:
-        chan.close()
+        if hasattr(chan, 'close'):
```
Comment (ploxiln) [inline bench/bench.py:260]: odd that this is needed

File: bench/bench.py [hunk 2]
```diff
@@ -283,15 +282,12 @@ def decomm():
...
-    tornado.options.define('secret_key', type=str, default='',
-                           help='AWS account secret key')
-    tornado.options.define('ami', type=str, default='ami-48fd2120',
+    # ubuntu 18.04 HVM instance store us-east-1
+    tornado.options.define('ami', type=str, default='ami-0938f2289b3fa3f5b',
                            help='AMI ID')
     tornado.options.define('ssh_key_name', type=str, default='default',
                            help='SSH key name')
-    tornado.options.define('instance_type', type=str, default='c3.2xlarge',
+    tornado.options.define('instance_type', type=str, default='c3.xlarge',
```
Comment (ploxiln) [inline bench/bench.py:290]: may I suggest `c5.large`

(which would also require changing the AMI to an HVM-EBS variant, I think ami-0ac80df6eff0e70b5 would do it, and maybe instance-create would then need a root-ebs-size parameter)

... but this could be updated in a separate PR, I could try that after this is merged

File: bench/bench.py (hunk 1 above)
Comment (mreiferson) [reply bench/bench.py]: agreed, missed that change, updated

File: bench/bench.py (hunk 2 above)
Comment (mreiferson) [reply bench/bench.py:289]: `c5.large` is VPC-only, so until we rewrite it to setup an entire VPC (not likely), this is Good Enough ™️ 

I didn't mean to change this from the original default, I'll update it.
